pkg/Microsoft.Private.PackageBaseline/stable.packages.props

```diff
@@ -1342,6 +1342,15 @@
     <StablePackage Include="System.Net.Http.WinHttpHandler">
       <Version>4.0.1</Version>
     </StablePackage>
+    <StablePackage Include="System.Private.DataContractSerialization">
```

Comment:
Yes I have rebuilt and ensured that the S.R.Serialization.Xml and S.R.Serialization.Json depend on the new S.Private.DataContractSerialization version 4.1.2. Is my best course of action to add a Revert commit to cancel the second change where I made changes to the stable packages list?